DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -92,10 +95,19 @@ public string StartDateYear
             get => new SelectElement(YearDropDown).SelectedOption.Text;
             set => new SelectElement(YearDropDown).SelectByText(value);
         }
+
         public string StartDateDay
         {
             get => new SelectElement(DayDropDown).SelectedOption.Text;
             set => new SelectElement(DayDropDown).SelectByText(value);
         }
+
+        public SettingsPage OpenSettings()
+        {
+            SettingsLnk.Click();
+            return new SettingsPage(driver);
+        }
+
+        public string CurrenrCurrencyFld => CurrentCurrency.Text;
```

Comment:
Or 'DepositCurrency', or:
```suggestion
        public string Currency => CurrentCurrency.Text;
```